strings/en_US/messages.po

```diff
@@ -3588,3 +3582,342 @@ msgstr "Manage Users"
+msgstr "It’s ok to run the Dojo by yourself, but we’d advise trying to find volunteers to help you."
+
+msgid "Your Dojo will grow and it’s always good to have extra hands."
+msgstr "Your Dojo will grow and it’s always good to have extra hands."
+
+msgid "You can find more tips on finding volunteers <a target=\"_blank\" href=\"https://hwf.zendesk.com/hc/en-us/articles/115003582283\">here.</a>"
+msgid "You can find more tips on finding volunteers <a target=\"_blank\" href=\"https://hwf.zendesk.com/hc/en-us/articles/115003582283\">here.</a>"
+
+msgid "Youth/community workers"
+msgstr "Youth/community workers"
+
+msgid "Primary or secondary teachers"
+msgstr "Primary or secondary teachers"
+
+msgid "IT professionals"
+msgstr "IT professionals"
+
+msgid "3rd level education students"
+msgstr "3rd level education students"
+
+msgid "Staff of venue"
+msgstr "Staff of venue"
+
+msgid "Youth Mentors (under 18)"
+msgstr "Youth Mentors (under 18)"
+
+msgid "Parents of attendees"
+msgstr "Parents of attendees"
+
+msgid "All global Dojos follow a common ethos and principles, such as always being free for children, and encouraging them to work together."
+msgstr "All global Dojos follow a common ethos and principles, such as always being free for children, and encouraging them to work together."
+
+msgid "All of these principles are  documented in the CoderDojo Charter."
+msgstr "All of these principles are  documented in the CoderDojo Charter."
+
+msgid "To become a verified Dojo you must agree to follow all principles in the CoderDojo Charter."
+msgstr "To become a verified Dojo you must agree to follow all principles in the CoderDojo Charter."
+
+msgid "It’s important that, as part of the CoderDojo community, you and your volunteers understand the CoderDojo Ethos."
+msgstr "It’s important that, as part of the CoderDojo community, you and your volunteers understand the CoderDojo Ethos."
+
+msgid "To make it easier to understand and implement in your Dojo we have created a short E-Learning module <a target=\"_blank\" href=\"/dashboard/profile/lms\">which you can find here.</a>"
+msgstr "To make it easier to understand and implement in your Dojo we have created a short E-Learning module <a target=\"_blank\" href=\"/dashboard/profile/lms\">which you can find here.</a>"
+
+msgid "You should encourage all volunteers to take the CoderDojo E-Learning modules."
+msgstr "You should encourage all volunteers to take the CoderDojo E-Learning modules."
+
+msgid "Venue Details"
+msgstr "Venue Details"
+
+msgid "Sign the Charter"
+msgstr "Sign the Charter"
+
+msgid "Review your application"
+msgstr "Review your application"
+
+msgid "Congratulations! Your Dojo application is being reviewed by a member of the CoderDojo Foundation team."
+msgstr "Congratulations! Your Dojo application is being reviewed by a member of the CoderDojo Foundation team."
+
+msgid "We will will respond to you within 48 hours, so hang tight while we check the information you have submitted."
```

Comment:
Our average is 30 hours or so even including weekends, leave at 48 hours. (though I get your last case but hopefully people won't be that anal!)